src/data/resources/filters.ts

```diff
@@ -42,6 +42,10 @@ export class Filters extends Base {
     filterId: string,
```

Comment:
nitpick(non-blocker): Consider adding [`@deprecated`](https://jsdoc.app/tags-deprecated.html) to the `Filter` class definition